src/components/PeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -1,70 +1,73 @@
 <template>
-  <v-card flat id="list-people-roles">
-    <ConfirmRemoveDialog
-      :dialog="dialog"
-      attach="#list-people-roles"
-      @confirm="emitRemovePerson(personId)"
-      @exit="dialog = false"
-    />
-
-    <!-- Summary Section -->
-    <div id="people-roles-summary" v-if="isSummary">
-      <!-- Summary Header -->
-      <div class="people-roles-summary-header" >
-        <v-icon>mdi-account</v-icon>
-        <label class="people-roles-title"><strong>People and Roles</strong></label>
-      </div>
-
-      <!-- Summary Warning -->
-      <div v-if="showErrorSummary" class="people-roles-invalid-message">
-        <span>
-          <v-icon color="blue darken-2">mdi-information-outline</v-icon>
-          This step is not complete.
-          <router-link id="router-link" :to="{ path: '/add-people-roles', query: { showErrors: true } }">
-            Return to this step to complete it.
-          </router-link>
-        </span>
-      </div>
-    </div>
-
+  <div flat id="list-people-roles" class="mt-0">
     <!-- List Display Section -->
-    <div id="people-roles-list">
+    <div id="people-roles-list" v-if="getPeopleAndRoles.length > 0">
       <!-- List Headers -->
-      <v-row class="people-roles-header list-item__subtitle" no-gutters>
+      <v-row class="people-roles-list-header list-item__subtitle pb-3" no-gutters>
         <v-col v-for="(title, index) in tableHeaders" :key="index">
           <span>{{ title }}</span>
         </v-col>
-        <!-- Spacer Column For Actions -->
-        <v-col sm="1" v-if="!isSummary"></v-col>
+        <!-- Spacer Column for Actions -->
+        <v-col lg="1"></v-col>
       </v-row>
 
       <!-- List Content -->
+
+      <!-- FUTURE: either show the item or show the edit component -->
```

Comment:
This is to possibly move the add/edit component inline (in the table) instead of replacing the table. This would be in a future ticket, so this is a placeholder for now.